server/handlers/events_streamer.go

```diff
@@ -475,3 +475,52 @@ func closeAdapterConnections(localMeshAdaptersLock *sync.Mutex, localMeshAdapter
 
 	return map[string]*meshes.MeshClient{}
 }
+
+// swagger:route POST /api/events EventsAPI idUIEventHandler
+// Handle sending UI-generated events to the notification center through the server.
+// responses:
+// 200:
+
+func (h *Handler) UIEventHandler(w http.ResponseWriter, req *http.Request, prefObj *models.Preference, user *models.User, provider models.Provider) {
+	fmt.Println("POST /api/events hit")
```

Comment:
Is this print statement intended for logging? If so, please replace it with the MeshKit logger. If not, please remove it.